rpcserver.go

```diff
@@ -6463,8 +6463,15 @@ func (r *rpcServer) GetTransactions(ctx context.Context,
 		endHeight = req.EndHeight
 	}
 
+	// To remain backward compatible, If the number of transactions was not
```

Comment:
nit: missing punctuation at end of sentence. And the word `If` probably shouldn't be capitalized in the middle of the sentence.